src/core_codemods/remove_assertion_in_pytest_raises.py

```diff
@@ -0,0 +1,158 @@
+
+    def _build_simple_statement_line(self, node: cst.BaseSmallStatement):
+        return cst.SimpleStatementLine(
+            body=[node.with_changes(semicolon=cst.MaybeSentinel.DEFAULT)]
+        )
+
+    def _remove_last_asserts_from_suite(self, node: Sequence[cst.BaseSmallStatement]):
+        assert_position = len(node)
+        assert_stmts = []
+        new_statement_before_asserts = None
+        for stmt in reversed(node):
+            match stmt:
+                case cst.Assert():
+                    assert_position = assert_position - 1
+                    assert_stmts.append(self._build_simple_statement_line(stmt))
+                case _:
+                    break
+        if assert_position > 0:
+            new_statement_before_asserts = node[assert_position - 1].with_changes(
+                semicolon=cst.MaybeSentinel.DEFAULT
+            )
+        return assert_stmts, assert_position, new_statement_before_asserts
+
+    def _remove_last_asserts_from_IndentedBlock(self, node: cst.IndentedBlock):
+        assert_position = len(node.body)
+        assert_stmts = []
+        new_statement_before_asserts = None
+        for simple_stmt in reversed(node.body):
+            match simple_stmt:
+                case cst.SimpleStatementLine(body=[*head, cst.Assert()] as body):
+                    assert_position = assert_position - 1
+                    if head:
+                        sstmts, s_pos, new_stmt = self._remove_last_asserts_from_suite(
+                            body
+                        )
+                        assert_stmts.extend(sstmts)
+                        if new_stmt:
+                            new_statement_before_asserts = new_stmt
+                            new_statement_before_asserts = simple_stmt.with_changes(
+                                body=[
+                                    *body[: s_pos - 1],
+                                    body[s_pos - 1].with_changes(
+                                        semicolon=cst.MaybeSentinel.DEFAULT
+                                    ),
+                                ]
+                            )
+                            break
+                    else:
+                        assert_stmts.append(simple_stmt)
+                    if new_statement_before_asserts:
+                        break
+                case _:
+                    if assert_position > 0:
+                        new_statement_before_asserts = node.body[assert_position - 1]
+                    break
+        assert_stmts.reverse()
+        return assert_stmts, assert_position, new_statement_before_asserts
+
+    def leave_With(
+        self, original_node: cst.With, updated_node: cst.With
+    ) -> Union[
+        cst.BaseStatement, cst.FlattenSentinel[cst.BaseStatement], cst.RemovalSentinel
+    ]:
+        if not self.filter_by_path_includes_or_excludes(
+            self.node_position(original_node)
+        ):
+            return updated_node
```

Comment:
you added a `test_exclude_line` but still it says not covered.... weird. 

Well actually, given this is a leave_with node, it's likely the same issue as the other codemods with this todo `# TODO: add filter by include or exclude that works for nodes`. So I think you can go ahead and remove the unit test and just add the todo line